test/cpp/api/modules.cpp

```diff
@@ -3501,8 +3529,22 @@ TEST_F(ModulesTest, PrettyPrintIdentity) {
 }
 
 TEST_F(ModulesTest, PrettyPrintFlatten) {
-  ASSERT_EQ(c10::str(Flatten()), "torch::nn::Flatten()");
-  ASSERT_EQ(c10::str(Flatten(FlattenOptions().start_dim(2).end_dim(4))), "torch::nn::Flatten()");
+  ASSERT_EQ(c10::str(Flatten()), 
+    "torch::nn::Flatten(start_dim=1, end_dim=-1)");
+  ASSERT_EQ(c10::str(Flatten(FlattenOptions().start_dim(2).end_dim(4))), 
+    "torch::nn::Flatten(start_dim=2, end_dim=4)");
+}
+
+TEST_F(ModulesTest, PrettyPrintUnflatten) {
+  ASSERT_EQ(
+      c10::str(Unflatten(UnflattenOptions(0, {2, 2}))),
+      "torch::nn::Unflatten(dim=0, unflattened_size={2, 2})");
+  ASSERT_EQ(
+      c10::str(Unflatten(UnflattenOptions(
+          "B",
+          {std::tuple<std::string, int64_t>{"B1", 2},
```

Comment:
tuple is used here. But nameshape_t now has pair, you can either use tuple or pair in nameshape_t, just need to make here consistent.

Reply:
I wonder how my tests built and passed with this, I'm fixing it